src/anomalib/deploy/export.py

```diff
@@ -113,7 +115,7 @@ def export(
         # Export model to onnx and convert to OpenVINO IR if export mode is set to OpenVINO.
         onnx_path = export_to_onnx(model, input_size, export_path)
         if export_mode == ExportMode.OPENVINO:
-            export_to_openvino(export_path, onnx_path)
+            export_to_openvino(export_path, onnx_path, metadata, input_size)
```

Comment:
would it be an idea to add `input_size` to `metadata`. Or would it be too much for a temporary solution?

Reply:
I was thinking of adding it to metadata earlier but we don't need this in the onnx model and we might forget to remove it later when we refactor the openvino part.